src/app/pages/system/acmedns/acmedns-list/acmedns-list.component.ts

```diff
@@ -0,0 +1,63 @@
+import { Component, OnInit } from '@angular/core';
+import { MatSnackBar } from '@angular/material';
+import { Router } from '@angular/router';
+import * as _ from 'lodash';
+import { WebSocketService } from '../../../../services/';
+import { T } from '../../../../translate-marker';
+
+@Component({
+  selector: 'app-acmedns-list',
+  template: `<entity-table [title]="title" [conf]="this"></entity-table>`
+})
+export class AcmednsListComponent implements OnInit {
+  public title = "ACME DNS Authenticators";
+  protected queryCall = "acme.dns.authenticator.query";
+  protected wsDelete = "acme.dns.authenticator.delete";
+  protected route_add: string[] = ['system', 'acmedns', 'add'];
+  protected route_success: string[] = [ 'system', 'acmedns' ];
+  protected entityList: any;
+
+  constructor(protected router: Router,
+    protected ws: WebSocketService, public snackBar: MatSnackBar) {
+  }
+
+  ngOnInit() {
+  }
+
+  public columns: Array < any > = [
+    { name: T('Authenticator'), prop: 'authenticator' },
+    { name: T('Name'), prop: 'name' }
+  ];
+
+  public config: any = {
+    paging: true,
+    sorting: { columns: this.columns },
+    deleteMsg: {
+      title: 'Name',
```

Comment:
Currently, the delete message is like 'Delete Name XXX'. You may want to change the title to be like 'ACME DNS Authenticator', then the delete message will be 'Delete ACME DNS Authenticator XXX'.

Reply:
Good catch!